src/pages/bug-bounty.tsx

```diff
@@ -889,7 +889,7 @@ export const ClientLogosSmall = graphql`
 export const query = graphql`
   query BugBountyPage {
     consensusBountyHunters: allConsensusBountyHuntersCsv(
-      sort: { order: DESC, fields: score }
```

Comment:
The sorting & filtering structure has changed in v5.

Reply:
This is way better